### DISCUSSION TITLE
Will the autocomplete feature be coming back?

### DISCUSSION THREAD

**Discussion by MrMarCode** [wavetermdev/waveterm#1021] [General]
I quite liked the integration with the fig autocomplete system. It's very useful! Is there any plan of bringing that back? 
Thanks!

**Comment by esimkowitz** [comment]
Hey @MrMarCode it's on our to-do list for sure, can't really provide a great timeline. It was pretty deeply integrated into our old architecture. When we made the jump to the new architecture, we lost some of the control over the prompt area. We are working out a plan to bring back some of these features like autocomplete, universal history, and dynamic prompts, but it probably won't land til end of this year/early next year.
